### ISSUE TITLE
After using the script, there is this prompt at the top of my interface. What is this? A witch?

### ISSUE THREAD

**Issue by two867** [kaimarks9/moonbix-bot#18]
![bn_2024092708eng](https://github.com/user-attachments/assets/1468a506-fbf7-4e91-b5ba-7c854a8bc2ce)

**Comment by two867** [comment]
> bro are u sure can read that message above?

yes, I have this prompt on both accounts. It only appeared after using the script. It never disappears. I don’t know why.

**Comment by XPacL** [comment]
Yeah, the same message right after i started the script

**Comment by two867** [comment]
> Yeah, the same message right after i started the script是的，在我启动脚本后立即收到相同的消息

Binance officials will not detect abnormalities, so cancel the short investment qualification.

**Comment by two867** [comment]
> Yeah, the same message right after i started the script

Binance officials will not detect anything unusual, disqualify the airdrop

**Comment by XPacL** [comment]
> Binance officials will not detect anything unusual, disqualify the airdrop ?

It may well be. Perhaps this is a kind of black mark of the account

**Comment by XPacL** [comment]
> bro the message above.. it just many players play that game so server is over loaded. isn't big deal. ban accounts or whatever u say..

My friends who don't use the script don't have such a message.
